### DISCUSSION TITLE
LangChain demo是否已经不再支持？无法找到对应的readme.md文件

### DISCUSSION THREAD

**Discussion by willwilling** [zai-org/ChatGLM3#757] [Show and tell]
![image](https://github.com/THUDM/ChatGLM3/assets/18168678/c9304108-7f40-46f3-bc27-e2633b710d52)

**Comment by willwilling** [comment]
![image](https://github.com/THUDM/ChatGLM3/assets/18168678/25d834fe-1da7-43cc-98df-ea54904f0a5b)
请问下这是什么？ SENIVERSE_KEY

**Comment by willwilling** [comment]
问题已解决。
但是找这个答案还是浪费了一些时间。
https://www.seniverse.com/可以获取SENIVERSE_KEY； 得是私钥

**Reply by zRzRzRzRzRzRzR** [reply]
支持，对应的readme移除了，因为全部升级到了lc 0.1.x
